common/test/acceptance/pages/lms/fields.py

```diff
@@ -163,26 +163,19 @@ def value_for_text_field(self, field_id, value=None, press_enter=True):
                 query.results[0].send_keys(u'\ue007')  # Press Enter
         return query.attrs('value')[0]
 
-    def value_for_textarea_field(self, field_id, value=None):
+    def set_value_for_textarea_field(self, field_id, value):
         """
-        Get or set the value of a textarea field.
+        Set the value of a textarea field.
```

Comment:
THANK YOU for fixing this. Having a combined getter/setter is good programming, but not deterministic enough for testing purposes. I've looked at this code a few times but never seemed to have enough time or context to upgrade it.